.github/workflows/deploy-site.yml

```
@@ -45,9 +45,9 @@ jobs:
       - name: Setup Pages
         uses: actions/configure-pages@v3
       - name: Upload artifact
-        uses: actions/upload-pages-artifact@v1
+        uses: actions/upload-pages-artifact@v2
```

A [v3](https://github.com/actions/upload-pages-artifact/pkgs/container/upload-pages-artifact/487379543?tag=4.0.0) version is also available. Do we want to update it while we are at it?
```suggestion
        uses: actions/upload-pages-artifact@v3
```